.ci_support/migrations/cuda120_ppc64le_aarch64.yaml

```diff
@@ -1,7 +1,8 @@
-migrator_ts: 1645421738
+migrator_ts: 1683090168
```

Comment:
Could be wrong about this, but it looks like the CUDA 11.2 arch migrator was dropped when adding the CUDA 12.0 arch migrator. Think we need both migrators here

Reply:
Fixed in 8963af7.